src/app/app.component.ts

```diff
@@ -38,8 +38,8 @@ export class AppComponent implements OnInit {
   coordinates: { latitude: number; longitude: number }[] = [];
   finalDestinationName = '';
   weatherCoordinates: any;
```

Comment:
This should not be of type `any`

Reply:
I created interface in model for that now